templates/event.php

```diff
@@ -134,14 +135,16 @@
 					<?php foreach ( $event_stats->rows() as $_locale => $row ) : ?>
 					<tr>
 						<td title="<?php echo esc_html( $_locale ); ?> "><a href="<?php echo esc_url( gp_url_join( gp_url( '/languages' ), $row->language->slug ) ); ?>"><?php echo esc_html( $row->language->english_name ); ?></a></td>
-						<td><?php echo esc_html( $row->created ); ?></td>
+						<td><a href="<?php echo esc_url( wp_make_link_relative( gp_url_join( get_the_permalink( $event_id ), 'translations', $row->language->slug ) ) ); ?>"><?php echo esc_html( $row->created ); ?></a></td>
+						<td><a href="<?php echo esc_url( wp_make_link_relative( gp_url_join( get_the_permalink( $event_id ), 'translations', $row->language->slug, 'waiting' ) ) ); ?>"><?php echo esc_html( $row->waiting ); ?></a></td>
```

Comment:
Currently, in these URLs we don't have the `glotpress` text just after the domain. `https://glotpress.test/events/{event-slug}/translations/{language-slug}/`. This update adds it `https://glotpress.test/glotpress/events/{event-slug}/translations/{language-slug}/`

```suggestion
						<td><a href="<?php echo esc_url( gp_url_join( gp_url( wp_make_link_relative( get_the_permalink( $event_id ) ) ), 'translations', $row->language->slug ) ); ?>"><?php echo esc_html( $row->created ); ?></a></td>
						<td><a href="<?php echo esc_url( gp_url_join( gp_url( wp_make_link_relative( get_the_permalink( $event_id ) ) ), 'translations', $row->language->slug, 'waiting' ) ); ?>"><?php echo esc_html( $row->waiting ); ?></a></td>
```

Reply:
@psrpinto running out of time today, could you help add a new `Urls` method for this? Thanks!